docs/index.md

```diff
@@ -17,4 +17,6 @@ description: |-
 
 ### Optional
```

Comment:
Change the ### Optional to ### Provider authentication

1. Local OAuth2 key file: 
2. OAuth2 parameters: